tox.ini

```diff
@@ -24,15 +24,7 @@ setenv =
     COVERAGE_FILE = {toxinidir}/.coverage
     COVERAGE_RCFILE = {toxinidir}/.coveragerc
 deps =
-    pyqt5==5.12.*
-    pyqtwebengine==5.12.*
-    oldest: scikit-learn~=0.22.0
```

Comment:
These `oldest` test are important because they check if the package still works on (selected) minimum requirements.

Reply:
I'll update them...

Reply:
Please do, I don't know what the oldest version should be here.